dev-requirements.txt

```diff
@@ -3,3 +3,4 @@ pep8==1.6.1
 pyflakes==0.8.1
 mccabe==0.3
 flake8==2.3.0
+selenium==2.44.0
```

Comment:
These are the packages that you only need for testing but not for running the server. Not a very big list. 
```
nose==1.3.4
mock==1.0.1
pyquery==1.2.8
```